app/src/backend/sessionRecording/getRecording.ts

```diff
@@ -24,7 +24,7 @@ const hasRecordingAccess = (
   if (recording.visibility === "only-me" && uid === recording.ownerId) return true;
 
   // TODO: We should ideally keep uids here instead of keeping emails.
-  if (recording.accessEmails.includes(email)) return true;
+  if (recording.accessEmails.includes(email) || uid === recording.ownerId) return true;
```

Comment:
Do we need it if Firebase security rules are already enforced? I would rather like to remove `hasRecordingAccess` function from client-side altogether.

Reply:
Only the case where the recording is a workspace recording and user is trying to access the recording url from a different workspace breaks when `hasRecordingAccess` is removed. This can't be avoided as according custom claims, user has access to recording and we don't know whether user is on the workspace with recording or not